.flake8

```
@@ -1,2 +1,3 @@
 [flake8]
 max-line-length = 160
+ignore = W604, W503
```

Why are these particular errors being ignored? I'm more in favour of ignoring on a per-line case generally.

---

W604 was giving a false positive for a string containing markdown. It is for Python 2 -> 3 migration which will never be applicable so it makes more sense to disable globally IMO.

W503 just was violated lots across the codebase and it just seems like something a formatter would fix anyway, so easier to just enable a formatter (ruff?) if you want one, or otherwise disable rules which are related to formatting.

---

Interesting! I wasn't seeing these errors raised by flake8, I'll try to reproduce locally after getting 3.11 running.

---

I managed to reproduce these flake8 failures with Python 3.12.2 locally. They're happening for code inside a string, so it looks a flake8 bug, and indeed the errors go away when I update flake8 to 7.0.0 in pyproject.toml - could you please take that approach instead of ignoring these codes.

---

Actioned.